uiowa/uiowa
# Headline field showing up in "Footer Contact Info" block

Closes #4331 


# How to test


1. checkout branch
2. `blt ds --site=sandbox.uiowa.edu` (Added a headline that says 'Headline' to the footer in prod for testing)
3.  Make sure the update hook runs (removes existing headline content)
4. Go to https://sandbox.local.drupal.uiowa.edu/block/16?destination=/node/1 and make sure there isn't a headline text field
